blocks/alert-bar-content-source-block/sources/alert-bar-collections.js

```diff
@@ -10,6 +10,7 @@ const fetch = ({ "arc-site": website }) => {
 		published: true,
 		size: 1,
 		...(website ? { website } : {}),
+		themes: "v2",
```

Comment:
This is a question for all of these changes, but why are we adding the param here to the content API call? Don't we need to update the cache key for the content source itself? If so, this should be in the `params` object above. I think there's a way to set a default value for it so we don't have to update every single call to these content sources as well. I'll try to dig up the relevant docs.

Reply:
Looks like the default value is only for content sources defined in JSON: https://docs.arcxp.com/alc?sys_kb_id=fdba431f476e1950a87626c2846d430c&id=kb_article_view. I guess the expectation is that you can set the defaults in the code if you're writing it out in JS. I'm now wondering what the expected change was when Matthew thought up this task. It seems like a bigger lift than necessary for something I'm 90% sure will be handled by the `d` parameter updating on the deployment being published.

Reply:
Adding them to the params is definitely possible.  It wasn't very clear to me what the point of the story was besides cache busting which could happen in two areas... one for Pagebuilder cache busting and the other for API call cache busting.  I could really add it to both for as much as this feels like a temporary solution to cover an edge case.

Thoughts @rmbrntt ?

Reply:
Decision point for me would be our confidence in the d= key breaking the cache, if high confidence then it sounds covered and we can keep this PR on hold but not close it (as a back-up option). If not high confidence then this PR is good to go. And worst case we just remove it after our sunset of 1.x date and all customers have moved. 

For now I'm thinking we can put this on hold, consider the ticket done in this sprint. And then keep the PR open but do not merge.

We have a dry run ticket where we should be able to test this out...

Reply:
The d param is applied to the client call, but this change applies to the serverside call that calls for the server to run this content source.  But there's a question as to this being a change to break the pagebuilder cache of the content source itself (which AFAIK is a very short term cache...)